.gitignore

```diff
@@ -10,6 +10,11 @@
 
 src/install
 
+# ignore vscode files
 .vscode
 
-build
```

Comment:
This should not be removed, why would we add the build in the git history

Reply:
This is not removed. For some reason, this line was just removed to a few lines below.